packages/block-library/src/navigation/edit.js

```diff
@@ -55,6 +55,8 @@ function Navigation( {
 
 	const { selectBlock } = useDispatch( 'core/block-editor' );
 
+	const blockProps = useBlockWrapperProps( { ref } );
```

Comment:
Does this ref need to be passed to the hook? It's not placed on the block's root element, but rather the contained `<NavigationPlaceholder />`.

Reply:
Oh, you're right. Maybe a rebase error. :)